test/integration/targets/vmware_guest_serial_port/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+# Test code for the vmware_guest_serial_port Operations.
+# Copyright: (c) 2019, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+- when: vcsim is defined
```

Comment:
Some functional tests cannot be run on govcsim (a vcenter API simulator), in this case, we start a block with `when: vcsim is not defined`.
There is no reason to do the opposite, we should be able to run the tests on a regular lab without any problem. Could you please remove the condition and the associated block.